zproject/default_settings.py

```diff
@@ -259,8 +259,11 @@
     ],
     # Limits total number of unauthenticated API requests (primarily
     # used by the public access option). Since these are
-    # unauthenticated requests, each IP address is a separate bucket.
+    # unauthenticated requests, each IPv4 address is a separate bucket.
+    # For IPv6, one bucket is used for each IPv6/64.
     "api_by_ip": [
+        # can be IPv4, tor-exit-node or IPv6/64.
```

Comment:
I think it's okay to not mention tor-exit-node, we'll get an IP address in the end.

Reply:
ok